t/plugin/batch-requests.t

```diff
@@ -1017,3 +1017,204 @@ GET /t
 passed
 --- no_error_log
 [error]
+
+
+
+=== TEST 24: customize uri, not found
+--- yaml_config
+plugin_attr:
+    batch-requests:
+        uri: "/foo/bar"
+--- config
+    location = /t {
+        content_by_lua_block {
+            local core = require("apisix.core")
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/batch-requests',
+                ngx.HTTP_POST,
+                [=[{
+                    "timeout": 100,
+                    "pipeline":[
+                    {
+                        "path": "/a"
+                    }]
+                }]=],
+                [=[[
+                {
+                    "status": 200
+                }
+                ]]=]
+                )
+
+            ngx.status = code
+            ngx.say(body)
+        }
+    }
+
+    location = /a {
+        content_by_lua_block {
+            ngx.status = 200
+        }
+    }
+--- request
+GET /t
+--- error_code: 404
+--- no_error_log
+[error]
+
+
+
+=== TEST 25: customize uri, found
```

Comment:
The `t/plugin/batch-requests.t` is long enough. Let's add test in a new file `t/plugin/batch-requests2.t`.